components/molecule/select/src/components/MoleculeInputSelect.js

```diff
@@ -0,0 +1,50 @@
+import {Children, cloneElement} from 'react'
+import cx from 'classnames'
+import PropTypes from 'prop-types'
+
+const BASE_CLASS = `sui-MoleculeSelect-inputSelect`
+const CLASS_CONTAINER = `${BASE_CLASS}-container`
+const CLASS_ARROW = `${BASE_CLASS}-arrow`
+const CLASS_ARROW_DOWN = `${CLASS_ARROW}--down`
+const CLASS_ARROW_UP = `${CLASS_ARROW}--up`
+
+const MoleculeInputSelect = props => {
+  const {onClick, iconArrowDown: iconArrow, isOpen, disabled, children} = props
+
+  const classNames = cx(CLASS_ARROW, {
+    [CLASS_ARROW_DOWN]: !isOpen,
+    [CLASS_ARROW_UP]: isOpen
+  })
+
+  const extendedChildren = () =>
+    Children.toArray(children)
+      .filter(Boolean)
+      .map((child, index) => cloneElement(child, {...props}))
```

Comment:
Small suggestion to avoid recreating a new props object as they should be inmutable.

```suggestion
      .map((child, index) => cloneElement(child, props))
```

Reply:
If I remove the dots, we break the "select" component. 

Check this out, this is how the component looks with the spread on the props. We have some options previously selected and the text "Select a ..."
<img width="618" alt="Captura de pantalla 2021-09-14 a las 12 22 17" src="https://user-images.githubusercontent.com/45047958/133240994-15a64137-03bb-4130-9ab5-1969cd8d300d.png">

And when we remove the spread, the component stops working and we can't select an option. 
<img width="1056" alt="Captura de pantalla 2021-09-14 a las 12 22 44" src="https://user-images.githubusercontent.com/45047958/133241117-dbeb973a-9711-45b1-aa3c-c564d066a68d.png">

Reply:
Fixed! Thanks